packages/react/src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+import type { Meta, StoryFn } from '@storybook/react';
+import { TrashFillIcon } from '@navikt/aksel-icons';
+
+import media from '../../../../../assets/img/card-media.png';
+import { Textfield } from '../form/Textfield';
+import { Select } from '../Select';
+import { Heading, Paragraph } from '../Typography';
+import { Button } from '../Button';
+import { Stack } from '../../../../../docs-components';
+
+import { Card } from '.';
+
+type Story = StoryFn<typeof Card>;
+
+export default {
+  title: 'Felles/Card',
+  component: Card,
+  parameters: {
+    layout: 'padded',
+  },
+} as Meta;
+
+export const Preview: Story = (args) => (
+  <Card
+    borderRadius='large'
+    border='subtle'
+    {...args}
+  >
+    <Card.Header>
+      <Paragraph size='medium'>Rolle 1</Paragraph>
+      <Button
+        variant='secondary'
+        color='danger'
+        size='small'
+        icon={<TrashFillIcon aria-hidden />}
+      >
+        Fjern
+      </Button>
+    </Card.Header>
+    <Card.Content>
+      <Select
+        label='Velg rolle'
+        options={[
+          { value: 'daglig leder', label: 'Dalig leder' },
+          { value: 'forretningsfører', label: 'Forretningsfører' },
+        ]}
+      />
+      <Textfield
+        size='small'
+        label='Fødsels- eller d-nummer'
+      />
+      <Textfield
+        size='small'
+        label='Etternavn'
+      />
+    </Card.Content>
+  </Card>
+);
+
+export const Variants: StoryFn<typeof Card> = () => {
```

Comment:
This story does not show all cards unless I go to the full page canvas.
Can you add a wrapper div so they stack, and we don't get a scrollbar on the docs page?
![image](https://github.com/digdir/designsystem/assets/14057205/bf780f46-e71b-4048-b63b-f363265e15e7)